product/views/ProvisionInfraTemplates.yaml

```diff
@@ -0,0 +1,95 @@
+#
+# This is an MIQ Report configuration file
+#   Single value parameters are specified as:
+#     single_value_parm: value
+#   Multiple value parameters are specified as:
+#     multi_value_parm:
+#       - value 1
+#       - value 2
+#
+
+# Report title
+title: Images
+
+# Menu name
+name: Images
+
+# Main DB table report is based on
+db: MiqTemplate
```

Comment:
wrong class, should be `ManageIQ::Providers::InfraManager::Template` (or `TemplateInfra`)